### ISSUE TITLE
Switch to better permalink for the data products

### ISSUE THREAD

**Issue by dfm** [dfm/k2-paper#23]
I think that this will probably have to wait for a revision. I think just serving it all up on BBQ will work for now.

**Comment by davidwhogg** [comment]
Yes, but for the journal version, let's cut a version to somewhere with more longevity.

Also I need you to help me build out hogg.nyc!

**Comment by dfm** [comment]
Also. The BBQ version doesn't even really work yet :-/ I'll fix that tomorrow morning... climbing and beer time now though!

:beers:
